pom.xml

```diff
@@ -1162,6 +1162,11 @@
                 <artifactId>spring-boot-autoconfigure</artifactId>
                 <version>${spring.boot.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.springframework.security</groupId>
+                <artifactId>spring-security-oauth2-client</artifactId>
+                <version>6.0.2</version>
```

Comment:
This version needs to sync with spring security. We define it as property above so use it.
`${spring.security.version}`. Also shouldn't you remove the spring-social dependency below?